src/ValueParsers/DecimalParser.php

```diff
@@ -56,6 +57,50 @@ private function getMath() {
 	}
 
 	/**
+	 * Splits the exponent from the scientific notation of a decimal number.
+	 *
+	 * @example splitDecimalExponent( '1.2' )  is  array( '1.2', 0 )
+	 * @example splitDecimalExponent( '1.2e3' )  is  array( '1.2', 3 )
+	 * @example splitDecimalExponent( '1.2e-2' )  is  array( '1.2', -2 )
```

Comment:
`@example` tags? Hm. I think this is hardly helpful. Better have this as test cases.